app/server/app.js

```diff
@@ -0,0 +1,73 @@
+const express = require('express');
+const mongoose = require('mongoose');
+const cors = require('cors');
+const passport = require('passport');
+const session = require('express-session');
+const cookieSession = require('cookie-session');
+const app = express();
+
+const config = require('./utils/config');
+const middleware = require('./utils/middleware');
+const logger = require('./utils/logger');
+
+const passportSetup = require('./passport/passport');
+const googleLogin = require('./passport/google');
+
+const authRouter = require('./routes/auth');
+const usersRouter = require('./controllers/users');
+const childrenRouter = require('./controllers/children');
+const samplesRouter = require('./controllers/samples');
+
+mongoose.set('debug', true);
```

Comment:
This could be another env var in `.env` or it could rely on `NODE_ENV == development`. WDYT? :)

Reply:
Thank you for the tip!! I implemented it directly then it's done :)
(cf commit [d6f5da4](https://github.com/TechLabs-Berlin/ss23-talk-a-palooza/pull/48/commits/d6f5da4def1e2f13f27fc5b5427c75cf0f4fc39c)